dali/daliadmin/daadmin.cpp

```diff
@@ -275,13 +275,13 @@ void set(const char *path,const char *val)
     root->setProp(tail,val);
     conn->commit();
     root->getProp(tail,newv);
```

Comment:
not new, but odd to get value back that just set, we should probably get rid of this. Now would be good time.